content/project/board.adoc

```diff
@@ -0,0 +1,42 @@
+---
+layout: simplepage
+title: "Jenkins Board"
+section: project
+---
+
+As per the Jenkins project link:/project/governance/#governance-board[governance] document, the project has a board consisting of three members who can be contacted confidentially by emailing jenkinsci-board@googlegroups.com
+
+## Current board members
+
+* link:/blog/authors/rtyler[R Tyler Croy] (tyler@monkeypox.org)
+* link:/blog/authors/kohsuke[Kohsuke Kawaguchi]  (kk@kohsuke.org)
+* Dean Yu (dean.yu@gmail.com)
+
+## Contacting the board
+The Jenkins governance board can be contacted via email: jenkinsci-board@googlegroups.com
+
+## Election process
+As of this writing, there has not yet been a board election since the Jenkins project took on its current governance structure.
+
+The next election is due in October 2019, see the link:/blog/2019/09/25/board-elections[announcement].
+
+The link:/project/board-election-process[Board Election] page provides the process.
+
+## Officers
```

Comment:
This document lists 3 out of 5 officers 😭

Reply:
Yes meant to call that out,
I don't like the duplication.

1 page is just focused on the executive team I believe and the other page describes the officers and their roles.

Think we should just remove the officers page and move it to the board?

Reply:
or my other thought was to remove who is currently the officer on the team leads page, and just have that it on the board page, and then add the other two officers to board marking as vacant?